packages/server/graphql/mutations/helpers/notifications/notifyMattermost.ts

```diff
@@ -0,0 +1,309 @@
+import segmentIo from '../../../../utils/segmentIo'
+import sendToSentry from '../../../../utils/sendToSentry'
+import {DataLoaderWorker} from '../../../graphql'
+import {
+  makeFieldsAttachment,
+  makeHackedFieldButtonValue,
+  makeHackedButtonPairFields
+} from './makeMattermostAttachments'
+import getSummaryText from './getSummaryText'
+import MeetingRetrospective from '../../../../database/types/MeetingRetrospective'
+import MeetingAction from '../../../../database/types/MeetingAction'
+import MeetingPoker from '../../../../database/types/MeetingPoker'
+import findStageById from 'parabol-client/utils/meetings/findStageById'
+import {phaseLabelLookup} from 'parabol-client/utils/meetings/lookups'
+import formatWeekday from 'parabol-client/utils/date/formatWeekday'
+import formatTime from 'parabol-client/utils/date/formatTime'
+import {toEpochSeconds} from '../../../../utils/epochTime'
+
+const okResponse: MattermostApiResponse = {
+  ok: true,
+  status: 0
+}
+
+const getWebhookForUserIdTeamId = async (
+  userId: string,
+  teamId: string,
+  dataLoader: DataLoaderWorker
+) => {
+  const mattermost = await dataLoader.get('mattermostBestAuthByUserIdTeamId').load({userId, teamId})
+  return mattermost?.webhookUrl
+}
+
+const notifyMattermost = async (
+  event: EventEnum,
+  webhookUrl: string,
+  userId: string,
+  teamId: string,
+  textOrAttachmentsArray: string | unknown[],
+  notificationText?: string
+) => {
+  const manager = new MattermostServerManager(webhookUrl)
+  const result = await manager
+    .postMessage(textOrAttachmentsArray, notificationText)
+    .catch((err) => new Error(`Mattermost error: ${err.message}`))
+  if (result instanceof Error) {
+    sendToSentry(result, {userId, tags: {teamId, event, webhookUrl}})
+    return result
+  }
+  if (result.ok) {
+    segmentIo.track({
+      userId,
+      event: 'Mattermost notification sent',
+      properties: {
+        teamId,
+        notificationEvent: event
+      }
+    })
+  }
+
+  return result
+}
+
+export const notifyWebhookConfigUpdated = async (
+  webhookUrl: string,
+  userId: string,
+  teamId: string,
+  dataLoader: DataLoaderWorker
+) => {
+  const {name: teamName} = await dataLoader.get('teams').load(teamId)
+  const message = `Integration webhook configuration updated for team \`${teamName}\``
+
+  const attachments = [
+    makeFieldsAttachment(
+      [
+        {
+          short: false,
+          value: message
+        }
+      ],
+      {
+        fallback: message
+      }
+    )
+  ]
+  return await notifyMattermost('meetingEnd', webhookUrl, userId, teamId, attachments)
+}
+
+export const startMattermostMeeting = async (
+  meetingId: string,
+  teamId: string,
+  dataLoader: DataLoaderWorker
+) => {
+  const meeting = await dataLoader.get('newMeetings').load(meetingId)
+  const webhookUrl = await getWebhookForUserIdTeamId(meeting.facilitatorUserId, teamId, dataLoader)
+  if (!webhookUrl) return okResponse
```

Comment:
since we're not using the `okResponse`, i'd probably just `return` to keep it simple